### PR TITLE
Add installation guide for namespaced mode

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref: https://github.com/pipe-cd/pipe/pull/1830

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/installation.md [hunk 1]
```diff
@@ -57,8 +57,42 @@ description: >
     --set-file secret.pipedKey.data=PATH_TO_PIPED_KEY_FILE \
     --set-file secret.sshKey.data=PATH_TO_PRIVATE_SSH_KEY_FILE
   ```
+
 Note: Be sure to set `--set args.insecure=true` if your control-plane has not TLS-enabled yet.
 
+See [values.yaml](https://github.com/pipe-cd/manifests/blob/master/manifests/piped/values.yaml) for the full values.
+
+### Installing on Kubernetes cluster in the namespaced mode
+The previous way requires installing cluster-level resources. If you want to restrict Piped's permission within the namespace as the same as Piped, this way is for you.
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/installation.md:66]: nits,
`The previous way requires installing Piped as cluster-level resource.`

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/installation.md:66]: For sure, but honestly I don't know exactly which is better. Most of Piped resources are namespace-level then only `Role/ClusterRole` and `RoleBindings/ClusterRoleBindings` can be different, that's why I thought "installing cluster-level resources" is kind of suitable for it. No strong opinion though 😃

Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation.md:66]: I get your point 👍 then keep it as is LGTM 🙆‍♀️ I don't have a strong objection to this

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/installation.md:66]: @khanhtc1202 Thanks!

**Pull request merged**